src/platform/datapath_epoll.c

```diff
@@ -364,6 +393,58 @@ CxPlatSocketSendInternal(
     _In_ BOOLEAN IsPendedSend
     );
 
+QUIC_STATUS
+CxPlatDataPathQuerySockoptSupport(
+    _Inout_ CXPLAT_DATAPATH* Datapath
+    )
+{
+    int Result;
+    socklen_t OptionLength;
+    QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
+
+    int UdpSocket = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+    if (UdpSocket == INVALID_SOCKET) {
+        int SockError = errno;
+        QuicTraceLogWarning(
+            DatapathOpenUdpSocketFailed,
+            "[data] UDP send segmentation helper socket failed to open, 0x%x",
+            SockError);
+        goto Error;
+    }
+
+#ifdef UDP_SEGMENT
```

Comment:
Practically, this function doesn't need to be called at all if this isn't set. That would also fix the code scanning errors below.